docs/ref/utils.txt

```diff
@@ -1011,10 +1011,10 @@ appropriate entities.
 For a complete discussion on the usage of the following see the
 :doc:`translation documentation </topics/i18n/translation>`.
 
-.. function:: gettext(message)
-
-    Translates ``message`` and returns it in a UTF-8 bytestring.
+The ``u`` prefix comes from the time there was a difference between unicode and
```

Comment:
The u prefix on the functions below comes from a difference in Python 2 between unicode and bytestrings. If your code doesn't support Python 2, use the functions without the u.